apps/docs/content/components/Label.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Label
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=6084-28401'
```

Comment:
This goes to Button page in Figma. Can we repoint to Label?

Reply:
Do you have the Figma branch updated? I just double-checked that and it goes to the Label page.

Reply:
Huh, yeah it goes there when i open in browser, but in app it loaded button. 🤷 .. guess it's fine and maybe my app is buggy.